events/lex.go

```diff
@@ -41,7 +41,7 @@ type LexDialogAction struct {
 	ResponseCard     *LexResponseCard  `json:"responseCard,omitempty"`
 }
 
-type Slots map[string]string
+type Slots map[string]interface{}
```

Comment:
Do I understand right, this is to support  explicit null slots? ex: `"slots": { "slot-name": null }`. Could this instead be `map[string]*string`? It'd help to see the test sample expanded, and an AWS documentation reference.

Reply:
You are right, according to the AWS Reference should be a string pointer.
Btw, this is because when I delegate the next action of the intent to Amazon, if the next required slot is empty string, Amazon avoid it, like the slot have been already filled.
Thanks.